spring-data-cassandra/src/test/java/org/springframework/data/cassandra/core/InsertOptionsUnitTests.java

```diff
@@ -30,13 +33,16 @@
 
 	@Test // DATACASS-250
 	public void shouldConfigureInsertOptions() {
+		Instant now = LocalDateTime.now().toInstant(ZoneOffset.UTC);
 
 		InsertOptions insertOptions = InsertOptions.builder()
 				.ttl(10)
+				.timestamp(now)
 				.withIfNotExists()
 				.build();
 
 		assertThat(insertOptions.getTtl()).isEqualTo(Duration.ofSeconds(10));
+		assertThat(insertOptions.getTimestamp() == now.toEpochMilli() * 1000);
```

Comment:
did you mean `assertThat(insertOptions.getTimestamp()).isEqualTo(now.toEpochMilli() * 1000);`?